Makefile

```diff
@@ -115,10 +120,18 @@ integration-test-kusama-call:
 integration-test-sibling-transfer:
 	RUST_LOG="xcm=trace,xcm-executor=trace" SKIP_WASM_BUILD= cargo test -p runtime-integration-tests -- polkadot_transfer --nocapture
 
+.PHONY: test-pallet-evm-precompile-assets-erc20
```

Comment:
consistency is important

`test-evm...` directly is better because we implies already pallet in `test-prices` etc